packages/web-components/fast-foundation/src/anchored-region/anchored-region.template.ts

```diff
@@ -2,7 +2,9 @@ import { html, ref, when } from "@microsoft/fast-element";
 import { AnchoredRegion } from "./anchored-region";
 
 export const AnchoredRegionTemplate = html<AnchoredRegion>`
-    <div class="region" part="region" ${ref("region")} style=${x => x.regionStyle}>
-        ${when(x => x.initialLayoutComplete, html`<slot></slot>`)}
-    </div>
+    <template ${ref("root")}>
```

Comment:
I don't think that you need a ref on the `<template>` - you can just use the `this` keyword to refer to the element in any method.

Reply:
Oh yeah.  Sweet.